plugins/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ option(ENABLE_PLUGIN_SWIOT "Enable SWIOT plugin" ON)
 option(ENABLE_PLUGIN_PQM "Enable PQM plugin" ON)
 option(ENABLE_PLUGIN_DATALOGGER "Enable DATALOGGER plugin" ON)
 option(ENABLE_PLUGIN_DAC "Enable DAC plugin" ON)
+option(ENABLE_PLUGIN_PLUTOPLUGIN "Enable PLUTOPLUGIN plugin" ON)
```

Comment:
Remove duplicate "Plugin" text to match all the other plugins. This needs to be done throughout the entire code, as it is an artifact from an improper completion of the plugin generator config json file.